Drivers/Inc/CAN.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CAN_H__
+#define CAN_H__
+
+typedef enum {
+	MOTOR_DRIVE = 0x221,
+    MOTOR_POWER = 0x222,
+    RESET = 0x223,
+    VELOCITY = 0x243

```

Comment:
These are the IDs that we're sending to the Motor Controller. The IDs that we'll be sending to the other car systems can be found here: https://docs.google.com/spreadsheets/d/11YWoMVZw8BFr8kyO4DIz0g-aIU_vVa0d-WioSRq85TI/edit#gid=0
under the Controls section.
Most of them are the forwarded messages that we receive from the motor controller.